pyndustric/compiler.py

```diff
@@ -539,15 +543,14 @@ def emit_sleep_syscall(self, node: ast.Call):
         arg = node.args[0]
         ms = self.as_value(arg)
 
-        sleep_label = _Label()
+        self.ins_append(f"wait {ms}")
+
+    def emit_asm_syscall(self, node: ast.Call):
```

Comment:
The problem with this is pyndustric may mangle some names. So I'm not sure how this is supposed to interact with the mangling.

Reply:
What do you mean?

Reply:
For example, when iterating over things, pyndustric will create a new variable name to index into the elements. The only way one can reference that with this `asm` instruction is by knowing how it's implemented and where.

I don't think we should add this. Adding missing features shouldn't be difficult, and "bugs to work around" should instead be fixed.

Did you have a particular case in mind to use this?

Reply:
Not anymore